rts/lra/test/README.adoc

```diff
@@ -1,57 +1,63 @@
+  which make possible to execute the test modules on supported runtimes
 * link:./basic[`basic`] contains integration tests which are written for testing
   the Narayana features of the
   https://github.com/eclipse/microprofile-lra[LRA specification]
-* link:./tck[`tck`] is a module which is responsible for execution of
+* link:./tck[`tck`] is a module which is responsible for the execution of
   https://github.com/eclipse/microprofile-lra/tree/master/tck[LRA TCK tests]
   with Narayana supported runtimes
-
-All tests are executed with help of https://arquillian.org[Arquillian] platform.
-Arquillian is responsible to launch the runtime and makes possible to execute
-the tests.
+* link:./crash[`crash`] hosts tests that simulate a crash of either lra-coordinator or
+  lra-participant or both. So far, only Wildfly is considered in this module as Quarkus
+  does not have an Arquillian adapter to run unmanaged tests
 
 == How to run integration tests
 
-The itegration tests which should be run on the application runtime
-are configured to be started with https://maven.apache.org/surefire/maven-failsafe-plugin/[Maven Failsafe Plugin].
+Integration tests are tested with
+https://maven.apache.org/surefire/maven-failsafe-plugin/[Maven Failsafe Plugin], using the target 'verify'.
 
-To run the tests the target `verify` has to be used.
-Single test with `-Dit.test=...`.
-Debug the Maven test execution with `-Dmaven.failsafe.debug`.
+Single test can be executed with `-Dit.test=...`
+To use Failsafe in debug mode, use `-Dmaven.failsafe.debug`
 
-The Failsafe Plugin link:../pom.xml#114[is configured] to run tests with names
+The Failsafe Plugin link:../pom.xml[is configured] to run tests with names
 `*IT.java`, `*Tests.java` and `*TestCase.java`.
 
 === Application runtimes
 
 The module is configured to run two runtimes - https://www.wildfly.org[WildFly]
-and link:https://quarkus.io[Quarkus]/link:https://thorntail.io[Thorntail].
+and link:https://quarkus.io[Quarkus].
 
-The **WildFly** runtime deploys the coordinator and the application with the client
-library at the same application server. The coordinator is deployed as
-a separated `war` archive, and the same the applications which packs together
-the LRA client library.
+When the **WildFly** runtime is used (through the 'arq' profile), two instances of Wildfly are initiated:
+ * The first instance is started using WildFly Maven Plugin, then lra-coordinator is deployed using the dedicated WAR
+ * The second instance is started using Arquillian, then an lra-participant is deployed
+Moreover, the testing class is hosted on a 'local' JVM, which runs separately from the servers.
 
-**Quarkus/Thorntail** splits the deployments to two separate JVMs.
-The Quarkus is started as Narayana LRA coordinator and
-Thorntail packs the client application with the LRA client library.
+Also, the configuration using **Quarkus** (through the 'arq.thorntail') runs lra-coordinator and lra-partipant(s)
```

Comment:
This line is mixing concepts, why would we use 'arq.thorntail' to configure quarkus - they are very different runtimes (one is based on a the WildFly application server whereas the other runs on a standard JVM (with bytecode augmentation).

Reply:
I see your point and I agree with you. I will rename `arq.thorntail` to `arq.quarkus`.

Reply:
But the profile starts thorntail not quarkus.

Reply:
This profile starts Quarkus (using a maven plugin) AND Thorntail (using Arquillian) but the "main" runtime is Quarkus as `lra-coordinator` runs in it.